hal/x86_fsp_tgl_stage1.ld.in

```diff
@@ -69,14 +79,22 @@ SECTIONS
 
     .fsp_s FSP_S_BASE :
     {
+        . += PRE_HEADER_FILL_SIZE;
```

Comment:
This is to align the authentication header keeping the same alignment as the source. E.g. if IMAGE_HEADER_SIZE is 256 and the FSP start address is e.g. at 0xE6000, the linker script will ensure that 0xF00 bytes fill the page at 0xE5000, then place the header at 0xE5F00 and the actual payload at 0xE6000.

Open to implementing alternative strategies, if provided by reviewer.

Reply:
if the page alignment constraint is from qemu flash size, I suggest padding the start of the final binary in the Makefile. This will simplify the linker script